src/mgsession.c

```diff
@@ -23,6 +22,9 @@
 #ifndef _MSC_VER
 #include <unistd.h>
 #endif
+#if !defined(_WIN32) || !defined(_MSC_VER)
+#include <stdalign.h>
+#endif
```

Comment:
Maybe unify these two.

Reply:
I've polished a bit, I'll add the CI job and merge when everything is compiling